internal/pkg/template/templates/workloads/services/rd-web/cf.yml

```diff
@@ -103,6 +103,9 @@ Resources:
                   Fn::GetAtt: [ {{$stackName}}, Outputs.{{$var}}]
               {{- end}}
               {{- end}}
+            {{- if .StartCommand }}
+            StartCommand: {{.StartCommand}}
```

Comment:
It seems like `start_command` exists in both code configuration and image configuration 🤔 if we just name it to be `StartCommand` how do users know which one is it? https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-apprunner-service-codeconfigurationvalues.html#cfn-apprunner-service-codeconfigurationvalues-startcommand

Reply:
We dont use code configuration in our CFN tempalte.

Exactly one of the configuration and image configuration needs to be used, and we are using the latter one.